release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -53,6 +53,10 @@ The installer now gathers serial console logs from the bootstrap and control pla
 
 For more information, see xref:../installing/installing-troubleshooting.adoc#installation-bootstrap-gather_installing-troubleshooting[Troubleshooting installation issues].
 
+[id="ocp-4-12-consistent-adddress-for-ironicapi"]
+==== Consistent IP address for Ironic API in bare-metal installations without a provisoning network  
+With this update, in bare-metal installations without a provisioning network, the Ironic API service is accessible through a proxy server. This proxy server provides a consistent IP address for the Ironic API service. If the Metal3 pod that contains `metal3-ironic` relocates to another pod, the consistent proxy address ensures constant communication with the Ironic API service.
```

Comment:
```suggestion
With this update, in bare-metal installations without a provisioning network, you can access the Ironic API service through a proxy server. This proxy server provides a consistent IP address for the Ironic API service. If the Metal3 pod that contains `metal3-ironic` relocates to another pod, the consistent proxy address ensures constant communication with the Ironic API service.
```

Reply:
Good point. I think the user can send API requests, however the Bare Metal Operator (BMO) is more likely to be using this API service to manage bare metal in the cluster AFAIK. There may be other components accessing the API also, so I would like to keep it a little vague, or at least focus on the change of API endpoint.